src/Dropdown.js

```diff
@@ -41,9 +41,11 @@ class Dropdown extends Component {
   renderTrigger () {
     const { trigger } = this.props;
 
+    const className = typeof trigger.className === 'string' ? `${trigger.className} dropdown-button` : 'dropdown-button';
+
     return React.cloneElement(trigger, {
       ref: (t) => (this._trigger = `[data-activates=${this.idx}]`),
-      className: 'dropdown-button',
+      className,
```

Comment:
and then in `test/DropdownButtonSpec.js` please add a `it('passes classname down to trigger')`